src/google/protobuf/io/tokenizer.cc

```diff
@@ -1002,9 +1002,19 @@ bool Tokenizer::ParseInteger(const std::string& text, uint64_t max_value,
 }
 
 double Tokenizer::ParseFloat(const std::string& text) {
+  double result;
+  GOOGLE_LOG_IF(DFATAL,
+         !TryParseFloat(text, &result))
```

Comment:
I hate side effected in LOG statements, do you mind switching this to the more straight forward:

```
if (!TryParseFloat(...)) {
  LOG(DFATAL) << ...
}
```

Reply:
done